examples/alanine_dipeptide_tps/AD_tps_2b_run_fixed.ipynb

```diff
@@ -133,9 +128,7 @@
   {
    "cell_type": "code",
    "execution_count": 8,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "sampler = paths.PathSampling(storage=paths.Storage(\"tps_nc_files/alanine_dipeptide_fixed_tps.nc\", \"w\", template),\n",
```

Comment:
This filename should probably don't include the directory to keep inline with the other notebooks
```suggestion
    "sampler = paths.PathSampling(storage=paths.Storage(\"alanine_dipeptide_fixed_tps.nc\", \"w\", template),\n",
```